README.md

```diff
@@ -53,7 +53,7 @@ Or if you want to run it from the source:
 
 ```sh
 $ yarn
-$ yarn start -- /path/to/archives
+$ yarn start /path/to/archives
```

Comment:
I've added this because you usually want to pass extra parameters and it's important they actually to the CLI and not to yarn.

```sh
yarn start /path/to/archives --dry-run
```

Reply:
Yarn will pass those arguments to the binary too, no need for the `--`, like it was necessary with npm

Reply:
Ah OK!